integrations/jira/main.py

```diff
@@ -75,6 +89,10 @@ async def handle_webhook_request(data: dict[str, Any]) -> dict[str, Any]:
         logger.info(f'Received webhook event for issue: {data["issue"]["key"]}')
         issue = await client.get_single_issue(data["issue"]["key"])
         await ocean.register_raw(ObjectKind.ISSUE, [issue])
+    elif "user" in data:
+        logger.info(f'Received webhook event for user: {data["user"]["accountId"]}')
+        user = await client.get_single_user(data["user"]["accountId"])
+        await ocean.register_raw(ObjectKind.USER, [user])
```

Comment:
are we also creating webhook for user?

Reply:
what about user deletion?

Reply:
Yes, we are. 
user_created, user_deleted and user_updated events. 

For delete, the response will return the user, with active prop as false